frontend/src/scenes/notebooks/Nodes/NotebookNodePerson.tsx

```diff
@@ -24,30 +25,35 @@ const Component = (props: NodeViewProps): JSX.Element => {
             {...props}
             href={urls.person(id)}
             resizeable={false}
+            compact={isShared}
         >
-            <div className="border bg-bg-light rounded">
-                <div className="p-4">
-                    {personLoading ? (
-                        <LemonSkeleton className="h-6" />
-                    ) : (
-                        <PersonHeader withIcon person={person} noLink />
+            {isShared ? (
```

Comment:
Would it be worth moving this logic to the `NodeWrapper` given it is repeated across many nodes? Maybe make shareable nodes opt in to future proof new nodes not being shareable without some consideration